scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -0,0 +1,93 @@
+            <th data-toggle='tooltip' data-container='body'
+             title="If parenthood is confirmed by peddy" style="width: 10%">Parenthood</th>
+            <th data-toggle='tooltip' data-container='body'
+             title="Downloadable CytoSure file" style="width: 5%">CGH</th>
+            <th data-toggle='tooltip' data-container='body'
+             title="Tissue origin for the sample" style="width: 13%">Tissue</th>
+            <th style="width: 5%"></th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for ind in case.individuals %}
+            <tr {% if ind.phenotype_human == 'affected' %} class="bg-danger-light" {% endif %}>
+              <td>{{ ind.display_name }}</td>
+              <td>
+                {% if ind.sex_human in ['female','male'] %}
+                  <i class="fa fa-{{ind.sex_human}}" aria-hidden="true"></i>
+                {% else %}
+                  {{ind.sex_human}}
+                {% endif %}
+                {% if ind.confirmed_sex %}
+                  <i class="fa fa-check"></i>
+                {% endif %}
+              </td>
+              <td><input name="age_{{ind.individual_id}}" type="number" step="0.1" min="0"
+                  class="form-control col-8" value="{{ind.age}}"></td>
+              <td>{{ ind.phenotype_human }}</td>
+              <td>{{ ind.analysis_type|upper }}</td>
+              <td>{{ ind.predicted_ancestry or 'N/A' }}</td>
+              <td>
+                {% if ind.confirmed_parent == True %}
+                  <i class="fa fa-check"></i>
+                {% elif ind.confirmed_parent == False %}
+                  <i class="fa fa-exclamation-circle"></i>
+                {% else %}
+                  N/A
+                {% endif %}
+              </td>
+              <td>
+                {% if ind.vcf2cytosure %}
+                <a href="{{ url_for('cases.vcf2cytosure', institute_id=institute._id,
+                      case_name=case.display_name, individual_id=ind.individual_id) }}" class="btn">
+                  <i class="fa fa-download"></i>
+                </a>
+                {% else %}
+                  N/A
+                {% endif %}
+                <td>
+                  <select class="form-control col-10" name="tissue_{{ind.individual_id}}">
+                    {% for key,tissue in tissues.items() %}
+                      <option value="{{tissue}}" {{ 'selected' if ind.tissue_type == tissue }}>{{tissue}}</option>
+                    {% endfor %}
+                  </select>
+                </td>
+                <td>
+                  <button class="btn btn-secondary btn-sm form-control" name="update_ind"
+                    type="submit" value="{{ ind.individual_id }}">Save</button>
+                </td>
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+  </div>
+</form>
+{% endmacro %}
+
+
+{% block scripts %}
```

Comment:
Huh? How does that work? Doesn't it have to be imported or inherited into the main page?!

Reply:
Well it could be a standalone html page, I didn't want to import more scripts then the one I needed for the popup I guess..